src/types/v125.ts

```diff
@@ -0,0 +1,359 @@
+import type { Result } from './support'
```

Comment:
Can you rework this file to only contain the changes required for power ?

Reply:
Ok

Btw, all the `src/types/v*.ts` filesstart with this line: 
~~~rust
import type { Result } from './support'
~~~
which is never used 
`'Result' is declared but its value is never read.`

Should I remove this line in all files?

Reply:
Yes I think you can safely remove this